projects/1.18/assets/materialis/materialis/lang/zh_cn.json

```
@@ -199,47 +235,145 @@
   "modifier.materialis.feeble": "无力",
   "modifier.materialis.feeble.description": "工具的击退效果降低",
   "modifier.materialis.feeble.flavor": "小心轻放",
+  "modifier.materialis.fluxburner": "能量燃烧室",
+  "modifier.materialis.fluxburner.description": "消耗能量提升采掘速度",
+  "modifier.materialis.fluxburner.flavor": "充能！",
+  "modifier.materialis.fluxburner.mining_speed": "充能采掘速度",
+  "modifier.materialis.fluxripper": "能量撕裂者",
+  "modifier.materialis.fluxripper.attack_damage": "充能攻击伤害",
+  "modifier.materialis.fluxripper.description": "消耗能量提升攻击伤害",
+  "modifier.materialis.fluxripper.flavor": "64核心！",
+  "modifier.materialis.fluxshield": "能量护盾",
+  "modifier.materialis.fluxshield.description": "有概率消耗能量保护你的工具不损失耐久度",
+  "modifier.materialis.fluxshield.flavor": "充满能量！",
   "modifier.materialis.freezing": "冷冻",
   "modifier.materialis.freezing.description": "对目标造成减速",
   "modifier.materialis.freezing.flavor": "像冰一样冷！",
   "modifier.materialis.galvanized": "电镀",
   "modifier.materialis.galvanized.description": "用2个升级槽换取1个能力槽",
   "modifier.materialis.galvanized.flavor": "不锈钢！",
   "modifier.materialis.half_life": "半衰期",
-  "modifier.materialis.half_life.3": "半衰期 III",
-  "modifier.materialis.half_life.4": "半衰期 IV",
+  "modifier.materialis.half_life.3": "半衰期II：第I章",
+  "modifier.materialis.half_life.4": "半衰期II：第II章",
   "modifier.materialis.half_life.description": "在耐久度高的时候会更快地消耗耐久，耐久度低的时候耐久则消耗得更慢",
   "modifier.materialis.half_life.flavor": "闻到了灰烬的味道",
+  "modifier.materialis.immersive_wrenching": "沉浸捶打",
+  "modifier.materialis.immersive_wrenching.description": "工具可以使多方块结构成型或者设置特定方块的面",
+  "modifier.materialis.immersive_wrenching.flavor": "普蒂斯·斯宾塞在此！",
   "modifier.materialis.inertia": "惯性",
   "modifier.materialis.inertia.description": "攻击速度降低，但攻击伤害提高",
   "modifier.materialis.inertia.flavor": "物体的性质就是如此",
-  "modifier.materialis.neptunes_blessing": "海王星的祝福",
-  "modifier.materialis.neptunes_blessing.description": "工具在水下的采掘速度提升，并增加对水栖生物造成的伤害",
+  "modifier.materialis.instakill": "即死",
+  "modifier.materialis.instakill.description": "攻击会立刻杀死敌人",
+  "modifier.materialis.instakill.flavor": "爆头！",
+  "modifier.materialis.instamine": "瞬挖",
+  "modifier.materialis.instamine.description": "挖掘会立级破坏方块",
+  "modifier.materialis.instamine.flavor": "来谈谈快速挖矿",
+  "modifier.materialis.lesser_psionizing_radiation": "Psi衰变",
+  "modifier.materialis.lesser_psionizing_radiation.description": "你的工具对psi离子有很强的传导性，适合施放术式",
```

```suggestion
  "modifier.materialis.lesser_psionizing_radiation.description": "你的工具对Psi子有很强的传导性，适合施放术式",
```
psion倒不一定是指离子，应该就是指一种粒子（尤其有可能是借用轻子的后缀？）
比如μ子（muon）和τ子（tauon）
不过ψ子不大符合psi这个模组本身，还是不推荐，但真的觉得这个可以的话应该也行？（）

下面这个是Psi模组灵感来源，《魔法科高校的劣等生》里的翻译法，也可以用作参考
```suggestion
  "modifier.materialis.lesser_psionizing_radiation.description": "你的工具对想子有很强的传导性，适合施放术式",
```